src/bootstrap/dist.rs

```diff
@@ -996,7 +996,7 @@ impl Step for Src {
             "src/tools/rustc-std-workspace-alloc",
             "src/tools/rustc-std-workspace-std",
             "src/librustc",
-            "src/libsyntax",
+            "src/librustc_ast",
```

Comment:
I don't understand why these are included here, is this left over from when `src/libproc_macro` had a dependency on these two crates? If so, we've shipping a lot of pointless rustc source code for a while now.

cc @Mark-Simulacrum @dtolnay

Reply:
They were added by https://github.com/rust-lang/rust/pull/58269 to fix https://github.com/rust-lang/rust/issues/58268

Reply:
Huh, but why these crates specifically? #58268 doesn't explain why those vs the entire source distribution (which is still available.. somehow, right?)

Reply:
My best guess: https://github.com/intellij-rust/intellij-rust/issues/2623 mentions compiler plugins.

I don't think the entire source distribution is otherwise available as a rustup component.

Reply:
Well, this list is probably missing a couple crates in terms of plugin development and nobody brought this up, maybe it's time to remove them. I'll bring this up on the existing issue.